### DISCUSSION TITLE
give me stars! ⭐️

### DISCUSSION THREAD

**Discussion by caneco** [Astrotomic/opendor.me#43] [Ideas]
Could be fun for users that have an account on opendor.me to **star/upvote** another profile.
And with that you can hight the top upvoted profiles in the homepage on the given week.

If Stars it's to similar to GitHub just name it to "**Knock**" – it would be fun saying:

> **Hey! Look how many _knocks_ I have on opendor.me.**

**Comment by Gummibeer** [comment]
I see a good intention here. But humans are evil and it will be another number people can use to do a popularity comparison and bully others.

I will keep it open - but want to note that this CAN get problematic. I will think about ways.
Possibly in combination with the "medals" maintainers can give contributors a way the community can do the same. #25

**Reply by SarahSibert** [reply]
I agree with you Tom, I like the fact that on OpenDor everyone is equal and you are only judged on your work alone.

**Reply by caneco** [reply]
I was hoping to knock some profiles…

It was just an idea… no worry 😁

**Reply by Gummibeer** [reply]
I'm thinking about something like "knocks" with a reason/description. So that it's not just "dumb" clicking but you have to say why you knock that person.
This doesn't have to be public but it's a clickbot prevention and at least moderators can see it and/or the knocked person itself.
So I don't want to fully reject it but implement it a different way.

**Reply by Wulfheart** [reply]
This would be fun though.
